cvat-data/webpack.config.js

```diff
@@ -50,20 +44,89 @@ const webConfig = {
     },
 };
 
-const nodeConfig = {
-    target: 'node',
-    mode: 'development',
-    devtool: 'source-map',
-    entry: './src/js/cvat-data.js',
+
+const worker_img = {
+    target: 'web',
+    mode: 'production',
+    entry: './src/js/unzip_imgs.js',
     output: {
         path: path.resolve(__dirname, 'dist'),
-        filename: 'cvat-data.node.js',
-        library: 'cvatData',
-        libraryTarget: 'commonjs',
+        filename: 'unzip_imgs.js',      
     },
-    stats: {
-        warnings: false,
+    devServer: {
+        contentBase: path.join(__dirname, 'dist'),
+        compress: false,
+        inline: true,
+        port: 3001,
     },
-};
+    module: {
+        rules: [{
+            test: /.js?$/,
+            exclude: /node_modules/,
+            use: {
+                loader: 'babel-loader',
+                options: {
+                    presets: [
+                        ['@babel/preset-env', {
+                            targets: {
+                                chrome: 58,
+                            },
+                            useBuiltIns: 'usage',
+                            corejs: 3,
+                            loose: false,
+                            spec: false,
+                            debug: true,
+                            include: [],
+                            exclude: [],
+                        }],
+                    ],
+                    sourceType: 'unambiguous',
+                },
+            },
+        }],
+    },
+}
+
+const worker_video = {
+    target: 'web',
+    mode: 'production',
+    entry: './src/js/decode_video.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'decode_video.js',      
+    },
+    devServer: {
```

Comment:
Do you actually need devServer in this configuration?

Reply:
I don't know. If I'll delete this code will page work?

Reply:
@ygnn123
Yes, I believe you can delete it.